bin/satellite-clone.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+from optparse import OptionParser
+import os
+import subprocess
+
+def yes_or_no(question):
+    while "the answer is invalid":
+        reply = str(raw_input(question+' (y/n): ')).lower().strip()
+        if reply[0] == 'y':
+            return True
+        if reply[0] == 'n':
+            exit()
+
+def main():
+    parser = OptionParser(usage="Usage: satellite-clone [options]\n" \
+				"Example: satellite-clone --start-at-task=\"Clean yum info\"")
+    parser.add_option("-v", "--verbose",
+                      action="store_true",
+                      dest="verbose",
+                      default=False,
+                      help="verbose output")
+    parser.add_option("--start-at-task",
+                      action="store", # optional because action defaults to "store"
+                      dest="start_at_task",
+                      help="Start at a specific task",)
+    parser.add_option("-i", "--interactive",
+                      action="store_true",
+                      dest="interactive",
+                      default=False,
+                      help="interactive mode with prompt for each task")
+    (options, args) = parser.parse_args()
+
+
+    DEFAULT_PRODUCTION_INSTALL_PATH = "/usr/share/satellite-clone"
+    DEFAULT_PLAYBOOK_FILE = DEFAULT_PRODUCTION_INSTALL_PATH + "/satellite-clone-playbook.yml"
+
+    yes_or_no("This will initiate satellite-clone. Do you want to proceed?")
+ 
+    ansible_args = [] 
+    if options.verbose:
+        ansible_args.append("-vvv")
+    if options.start_at_task:
+        ansible_args.append("--start-at-task \"{0}\"".format(options.start_at_task))
+    if options.interactive:
+        ansible_args.append("--interactive")
+
+    if not os.path.isdir(DEFAULT_PRODUCTION_INSTALL_PATH):
+         print("It looks like satellite-clone has not been installed properly, " \
+               "{0} does not exist.".format(DEFAULT_PRODUCTION_INSTALL_PATH))
+         exit()
+
+    os.environ["ANSIBLE_CONFIG"] = os.path.join(DEFAULT_PRODUCTION_INSTALL_PATH, 'ansible.production.cfg')
+
+    inventory_path = os.path.join(DEFAULT_PRODUCTION_INSTALL_PATH, 'inventory')
+    ansible_playbook = "ansible-playbook -i {0} {1} {2}".format(inventory_path, 
```

Comment:
Also note that ```subprocess.Popen``` accepts a list which gives you shell quoting for free providing better security.